docs/user-quick-start.md

```diff
@@ -149,22 +149,20 @@ enough resources:
    for the details.
    Tested with version v1.12.0.
 
-1. Install the `virtctl` tool. See
-   [virtctl install](https://kubevirt.io/quickstart_minikube/#virtctl)
-   for the details.
-   Tested with version v1.0.1.
+1. Install the `virtctl` tool.
 
-   ```
-   curl -L -o virtctl https://github.com/kubevirt/kubevirt/releases/download/v1.0.1/virtctl-v1.0.1-linux-amd64
-   ```
-
-   After download completes for `virtctl` issue these commands.
+   Working Example:
 
    ```
-   chmod +x virtctl
-   sudo install virtctl /usr/local/bin
+   curl -L -O https://github.com/kubevirt/kubevirt/releases/download/v1.2.0/virtctl-v1.2.0-linux-amd64
+   sudo install virtctl-v1.2.0-linux-amd64 /usr/local/bin/virtctl
+   sudo rm virtctl-v1.2.0-linux-amd64
```

Comment:
Why do we need sudo to remove the downloaded file?

I think the previous example was nicer - it used "-o virtctl" so the downloaded file is already named correctly. This simplifies the next steps.

How about this?

```
curl -L -o virtctl https://github.com/kubevirt/kubevirt/releases/download/v1.2.0/virtctl-v1.2.0-linux-amd64
sudo install virtctl /usr/local/bin/
rm virtctl
```

Reply:
Okay you mean giving a name to the downloaded file itself.. yeah that's better. Changing it that way